components/dialog/test/dialog-fullscreen.visual-diff.js

```diff
@@ -88,6 +88,43 @@ describe('d2l-dialog-fullscreen', () => {
 					await visualDiff.screenshotAndCompare(page, this.test.fullTitle());
 				});
 
+				it('fullscreen-within-on', async function() {
```

Comment:
Same tests as in `d2l-dialog`, making sure toggling the fullscreen-within class functions as expected.